cdap-ui/app/features/tracker/services/my-tracker-api.js

```diff
@@ -67,7 +67,10 @@ function myTrackerApi(myCdapUrl, $resource, myAuth, myHelpers, UI_CONFIG) {
     demotePreferredTags: myHelpers.getConfig('POST', 'REQUEST', tagsPath + '/demote', false, { suppressErrors: true }),
     promoteUserTags: myHelpers.getConfig('POST', 'REQUEST', tagsPath + '/promote', false, { suppressErrors: true }),
     validatePreferredTags: myHelpers.getConfig('POST', 'REQUEST', tagsPath + '/validate', false, { suppressErrors: true }),
+    getEntityTags: myHelpers.getConfig('GET', 'REQUEST', tagsPath + '/:entityType/:entityId', false, { suppressErrors: true }),
+    addEntityTag: myHelpers.getConfig('POST', 'REQUEST', tagsPath + '/promote/:entityType/:entityId', false, { suppressErrors: true }),
```

Comment:
adding entity tag does not mean promoting it as a preferred tag.. so calling promote here is wrong